tanner/utils/logger.py

```diff
@@ -4,16 +4,22 @@
 
 class Logger:
     @staticmethod
-    def create_logger(filename, logger_name):
-        log_filename = filename
+    def create_logger(debug_filename, err_filename, logger_name):
         logger = logging.getLogger(logger_name)
         logger.setLevel(logging.DEBUG)
-        handler = logging.handlers.RotatingFileHandler(log_filename, encoding='utf-8')
-
         formatter = logging.Formatter(fmt='%(asctime)s %(levelname)s:%(name)s:%(funcName)s: %(message)s',
                                       datefmt='%Y-%m-%d %H:%M')
-        handler.setFormatter(formatter)
 
-        logger.addHandler(handler)
+        # DEBUG log to 'tanner.log'
+        debug_log_handler = logging.handlers.RotatingFileHandler(debug_filename, encoding='utf-8')
+        debug_log_handler.setFormatter(formatter)
```

Comment:
Ah ok, it's different from what I assumed. This is caused by the log level hirachy. I'll inspect how to handle it properly.